cmd/generic-handlers.go

```diff
@@ -580,6 +580,12 @@ func setBucketForwardingHandler(h http.Handler) http.Handler {
 					r.URL.Scheme = "https"
 				}
 				r.URL.Host = getHostFromSrv(sr)
+				w.Header().Del("Vary")
```

Comment:
Maybe just remove all headers in w before sending the request to the other server ?

Reply:
No we cannot remove all headers @vadmeste - some are needed - we should only remove for each context.

Reply:
Actually you are right, removed all headers @vadmeste